src/pages/Home.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import logo from '../assets/1logo.png'
+import { FaLinkedin } from "react-icons/fa6";
+import { RiInstagramFill } from "react-icons/ri";
+import { BsDiscord } from "react-icons/bs";
+import scales from '../assets/scales.png'
+import Foother from '../components/Footer'
+export default function Home() {
+  return (
+    <>
+      <div className='max-w-6xl mx-auto'>
+        <div className='flex items-center flex-col justify-center h-screen '>
+          <div className='flex items-center'>
+            <h1 className='text-6xl font font-bold text-[#3e3e3e]'>Dope Devs</h1>
+            <img src={logo} alt="logo" className='h-[60px]' />
+          </div>
+          <p className='text-[#5a5a5a] font-semibold my-4 text-xl font'>Crafting coders, the dope way!</p>
+          <div>
+            <ul className='flex md:space-x-6 text-[#3e3e3e] cursor-pointer space-x-2 '>
+              <li className='flex items-center lg:text-2xl md:text-xl border border-gray-400 px-1 rounded-md hover:scale-110 duration-200'><BsDiscord className='mx-2' /> <a href="https://discord.gg/wVaySB5H">Discord </a></li>
+              <li className='flex items-center lg:text-2xl md:text-xl  border border-gray-400 px-1 rounded-md hover:scale-110 duration-200'><RiInstagramFill className='mx-2' /> <a href="https://www.instagram.com/dope_devs/">Instagram </a></li>
+              <li className='flex items-center  lg:text-2xl md:text-xl  border border-gray-400  px-1 py-2 rounded-md hover:scale-110 transition duration-200'><FaLinkedin className='mx-2' /> <a href="https://www.linkedin.com/company/dope-devs/">LinkedIn </a></li>
+            </ul>
+          </div>
+        </div>
+
+        <div>
+          <h1 className='text-[#3e3e3e] text-2xl font-bold text-center uppercase my-4 font'>Unveiling the Passion Behind <span className='underline'> Dope Devs</span>🔥 </h1>
+          <p className='text-[#5a5a5a] font md:mx-24 mx-3'>Welcome to Dope Devs, a dynamic community that celebrates the art of web design, development, databases, and apps. Rooted in the philosophy of "Crafting coders, the dope way," we are a vibrant collective of individuals at all stages of their coding journey. <strong>Our mission</strong> revolves around creating an inclusive space where web technologies are not just learned but enjoyed. We believe in the power of collaboration, shared learning, and coding together to elevate each other's skills. Whether you're a seasoned web designer, a proficient developer, or someone exploring the realms of databases and apps, you'll find a welcoming home in our community.</p>
```

Comment:
Rewrite it and take help from Tayyaba - put it right after the social links without too much spacing.